composer.json

```diff
@@ -17,17 +17,14 @@
     },
     "config": {
         "sort-packages": true,
-        "platform": {
-            "php": "8.0.99"
-        },
         "allow-plugins": {
             "dealerdirect/phpcodesniffer-composer-installer": true
         }
     },
     "extra": {
     },
     "require": {
-        "php": "~8.0.0 || ~8.1.0 || ~8.2.0",
+        "php": "^8.0",
```

Comment:
We don't know if PHP 8.3 will break BC, so we cannot be this lax with the range.

The range should be `~8.1.0 || ~8.2.0`